src/views/Home.vue

```diff
@@ -1,17 +1,20 @@
 <template>
   <div class="home">
-    <messages-container />
+    <!-- <messages-container /> -->
+    <mail-container />
```

Comment:
The mail-container should be inside the messages-container, right next to the ConversationMessage component. 

Doing this will avoid using two different inbox components. We do not want 2 different inbox components as they are the same for both the job banks(your component) and the Virtual Concierge(ConversationMessage component).